build/CoreGraphics/dll/CoreGraphics.def

```diff
@@ -276,13 +292,13 @@ LIBRARY CoreGraphics
         CGFontGetUnitsPerEm
 
         ; private exports below
-        CGFontFitChars
-        CGFontGetGlyphs
-        CGFontGetGlyphsForUnichars
-        CGFontSetFTFontSize
-        CGFontWrap
-        _CGFontLock
-        _CGFontUnlock
+        ; CGFontFitChars
+        ; CGFontGetGlyphs
+        ; CGFontGetGlyphsForUnichars
+        ; CGFontSetFTFontSize
+        ; CGFontWrap
+        ; _CGFontLock
+        ; _CGFontUnlock
```

Comment:
Assuming these are not gone for good?  Do we have an issue to track putting these back in?